app/controllers/games/list.js

```diff
@@ -6,5 +6,10 @@ export default Ember.Controller.extend({
     return !Ember.isNone(this.get('session.user'));
   }.property('session.user'),
 
-  cannotCreateGame: Ember.computed.not('canCreateGame')
+  cannotCreateGame: Ember.computed.not('canCreateGame'),
+
+  /** @property {Array<Game>} all the games that haven't been ended */
+  joinableGames: Ember.computed('model.[].status', function () {
```

Comment:
I'm a little fuzzy on this syntax. I've only ever used `.@each` for depending on properties of array contents. What is the difference here?